### PR TITLE
Update bookings-in-outlook.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the domains in Default Sharing Policy

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9202

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/bookings/bookings-in-outlook.md [hunk 1]
```diff
@@ -65,9 +65,14 @@ For more information, see the [Bookings with me Microsoft 365 Roadmap item](http
      Get-SharingPolicy -Identity "Default Sharing Policy" | fl Domains 
    ```
 
-    "Anonymous:CalendarSharingFreeBusyReviewer"" should be one of the domains in the response.
-
-   To enable anonymous sharing, use the following command.
+    "Anonymous:CalendarSharingFreeBusySimple": Share free/busy hours only.
+    "Anonymous:CalendarSharingFreeBusyDetail": Share free/busy hours, subject and location.
+    "Anonymous:CalendarSharingFreeBusyReviewer": Share free/busy hours, subject, location and the body of the message or calendar item.
+    "Anonymous:ContactsSharing": Share contacts only.
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/bookings-in-outlook.md]:
```suggestion
You should have one of the following domains in the response:

    "Anonymous:CalendarSharingFreeBusySimple": Share free/busy hours only.
    
    "Anonymous:CalendarSharingFreeBusyDetail": Share free/busy hours, subject, and location.
    
    "Anonymous:CalendarSharingFreeBusyReviewer": Share free/busy hours, subject, location, and the body of the message or calendar item.
    
    "Anonymous:ContactsSharing": Share contacts only.
```

File: microsoft-365/bookings/bookings-in-outlook.md [hunk 2]
```diff
@@ -65,9 +65,14 @@ For more information, see the [Bookings with me Microsoft 365 Roadmap item](http
...
+    
+   You should have one of the above domains in the response.
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/bookings-in-outlook.md]:
```suggestion
```

File: microsoft-365/bookings/bookings-in-outlook.md [hunk 3]
```diff
@@ -65,9 +65,14 @@ For more information, see the [Bookings with me Microsoft 365 Roadmap item](http
...
+
+   To enable anonymous sharing, use the following command or alter **Add** section with the domain that you would like to add.
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/bookings-in-outlook.md]:
```suggestion
   To enable anonymous sharing, use the following command or alter the **Add** section with the domain that you would like to add.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by tiburd** [comment]
@msbemba @JohanFreelancer9 @yogkumgit 
Hi!
This repo now has build validation checks turned on - see above.
Note that validation must pass for the PR to be merged:
image
For your questions, please send me email. 
Thank you!

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. thanks

**Comment by tiburd** [comment]
@get-itips @dariomws @tseward 
Hi! Can you please review the changes to the PowerShell blocks in this PR?
Thanks!

**Review by ghost — APPROVED** [review]

File: microsoft-365/bookings/bookings-in-outlook.md [hunk 4]
```diff
@@ -57,17 +57,25 @@ For more information, see the [Bookings with me Microsoft 365 Roadmap item](http
...
+You should have one of the following domains in the response:
 
-   To enable anonymous sharing, use the following command.
+    Anonymous:CalendarSharingFreeBusySimple: Share free/busy hours only.
+    
+    Anonymous:CalendarSharingFreeBusyDetail: Share free/busy hours, subject, and location.
+    
+    Anonymous:CalendarSharingFreeBusyReviewer: Share free/busy hours, subject, location, and the body of the message or calendar item.
+    
+    Anonymous:ContactsSharing: Share contacts only.
```
Comment (ekaplanb) [inline microsoft-365/bookings/bookings-in-outlook.md:76]: Share contact only doesn't allow viewing the Calendar  Freebusy status of the organizer, why added?

**Comment by yogkumgit** [comment]
@msbemba

**Comment by get-itips** [comment]
I think this PR https://github.com/MicrosoftDocs/microsoft-365-docs/pull/9473 replaces this one, @msbemba please confirm, if positive, this PR should be closed.

**Comment by msbemba** [comment]
Closing this PR as we have another one replacing this #9473 https://github.com/MicrosoftDocs/microsoft-365-docs/pull/9473

**Pull request closed without merging**